### PR TITLE
Mainline Medieval Stuff mod, part 2

### PR DESCRIPTION
**Pull request by Rail-Runner**
Move some stuff from Medieval Stuff mod, mostly weapons.
Must be merged after part 1 ( #14601 ) to avoid errors.

Full list of moved items:

Melee:

arming sword (+fake) (+recipe) (+itemgroups)
longsword (+fake) (+recipe) (+itemgroups)
estoc (+fake) (+recipe) (+itemgroups)
halberd (+fake) (+recipe) (+itemgroups)
scimitar (+fake) (+recipe) (+itemgroups)
glaive (+recipe) (+itemgroups)
naginata (+recipe) (+itemgroups)
jian (+fake) (+recipe) (+itemgroups)
qiang (+recipe) (+itemgroups)

Ranged:

fire lance (+recipe) (+itemgroups)

Ammo:

fire lance charge (+recipe) (+itemgroups)
fire lance shot (+recipe) (+itemgroups)

### CODE REVIEW

**Comment by chaosvolt** [comment]
Merge conflicts. That bodes unwell. ;w;

**Comment by Darkyhard** [comment]
What causes the conflict and how do I check it myself?

**Comment by chaosvolt** [comment]
Usually occurs when the master version merged a PR that makes major changes to the same files the open PR is editing. The few times it's happened to me, I usually find it more of a clusterfuck to resolve than it should be. @_@

**Comment by Darkyhard** [comment]
Well, I know what causes this, but how do I actually understand which lines cause merge conflict?

**Comment by Zireael07** [comment]
Most git clients have a tool to show you what causes conflicts. Try SourceTree or some other GUI tool of your choice.

**Comment by chaosvolt** [comment]
If I recall, I had to try and merge the latest master version into the branch via Git Shell, then it told me the files that caused the issue.

However, I couldn't find any tool in the normal Github client to actually resolve the merge, so I had to set the branch's version of the file aside, replace it with the version from the freshly-updated master branch, then re-add the desired changes piece by piece. >.<

**Comment by Darkyhard** [comment]
It looks like renaming "grey" color to "gray" caused the conflict.

**Comment by chaosvolt** [comment]
Oh that is weird. When did that happen? ;w;

**Comment by Darkyhard** [comment]
It is caused by this PR: #14591

File: data/json/items/tools.json [hunk 1]
```diff
@@ -5451,8 +5475,106 @@
     "use_action": { "type": "knife" },
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 9]]
+  },{
+    "id": "scimitar",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. Designed for slashing, and quite deadly against unarmored targets.",
```
Comment (illi-kun) [inline data/json/items/tools.json]: Double spaces after periods please.

Comment (Darkyhard) [inline data/json/items/tools.json]: Done.

**Pull request closed without merging**